frontend/src/features/admin-form/create/workflow/components/WorkflowContent/EditStepBlock/RespondentBlock.tsx

```diff
@@ -31,8 +35,23 @@ export const RespondentBlock = ({
     formState: { errors },
     register,
     getValues,
+    control,
   } = formMethods
 
+  const isTest = process.env.NODE_ENV === 'test'
```

Comment:
/nit could you also put the note here to remove the flag like you did [here](https://github.com/opengovsg/FormSG/pull/7620/files#diff-b7b4ae2945123935752455bbf96f841eb7d4bdaa1800eed17533bf5b25988401L323)? It made it easy to reference and cleanup.